From: "Diego M. Rodriguez" <diego@moreda.io>
Date: Wed, 29 Dec 2021 16:19:05 +0100
Subject: Disable jupyter-based documentation examples

Remove the "docs/examples" references in the documentation, along with
the requirement of "nbsphinx", in order to allow the documentation
package to be built while work in #1001283 is ongoing.

See also: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1001283
---
 docs/conf.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

Index: pythran/docs/conf.py
===================================================================
--- pythran.orig/docs/conf.py	2023-01-08 14:15:02.933710795 +0100
+++ pythran/docs/conf.py	2023-01-08 14:15:02.925710714 +0100
@@ -29,7 +29,6 @@
 
 
     MANUAL
-    EXAMPLES
     CLI
     SUPPORT
     DEVGUIDE
@@ -126,7 +125,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['nbsphinx',]
+extensions = []
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
